### ISSUE TITLE
Not able to create bucket

### ISSUE THREAD

**Issue by junjie-landing** [Snowflake-Labs/terraform-snowflake-storage-integration-aws#20]
```
│ Error: error creating S3 bucket ACL for snowflake-to-athena-reg-staging-staging-bucket: AccessControlListNotSupported: The bucket does not allow ACLs
│ 	status code: 400, request id: B601YSZHCBW5K5AT, host id: VJI/Z69a1uW/icLR8/mhyTkOyiEPXPz2PybAOLoW+tLsb/Yl0sIM1tn7dlmf9mL/cdWcguW+v4w=
```
<img width="899" alt="image" src="https://github.com/Snowflake-Labs/terraform-snowflake-storage-integration-aws/assets/81201171/31a44798-c22b-4f5b-aa91-267a0547d3ff">

**Comment by sfc-gh-pkommini** [comment]
Should be fixed in #22.

**Comment by sfc-gh-pkommini** [comment]
@junjie-landing Release `v0.2.8` should not have this issue. Let me know if you have any issues.
